### ISSUE TITLE
Keras: not showing BatchNormalization layer

### ISSUE THREAD

**Issue by RezuanaHaque** [lutzroeder/netron#971]

**Comment by lutzroeder** [comment]
@RezuanaHaque please share a sample file and how it was created.

**Comment by RezuanaHaque** [comment]
This is my model:
```
model = Sequential()
model.add(Conv2D(16, (3, 3), input_shape=(height, width, 3), kernel_regularizer=l2(0.001)))
model.add(Activation('relu'))
# BatchNormalization()
model.add(MaxPooling2D(pool_size=(2, 2), strides=(2)))

BatchNormalization()

model.add(Conv2D(32, (3, 3), kernel_regularizer=l2(0.001)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2), strides=(2)))

BatchNormalization()
model.add(Dropout(.5))

model.add(Conv2D(32, (3, 3), kernel_regularizer=l2(0.001)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2), strides=(2)))

BatchNormalization()
model.add(Dropout(.3))

model.add(Conv2D(64, (3, 3), kernel_regularizer=l2(0.001)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2), strides=(2)))

BatchNormalization()
model.add(Dropout(.3))

model.add(Flatten())

model.add(Dense(256))
model.add(Activation('relu'))

model.add(Dropout(.5))

model.add(Dense(4,activation='softmax'))

```
This is the netron model:
![image](https://user-images.githubusercontent.com/78899126/190892614-1eb2e861-ad26-4c2e-b846-b0d375e5bb63.png)

**Comment by lutzroeder** [comment]
The script doesn't add `BatchNormalization()` to the model?

**Comment by RezuanaHaque** [comment]
No, I shared the code along with the model netron is showing

**Comment by swapnilsayansaha** [comment]
@RezuanaHaque your code is wrong. Look at your code carefully. Your code is not adding batch norm to the model. Correct way to add batch norm is: ```model.add(BatchNormalization())```
